spec/client_spec.rb

```diff
@@ -3,6 +3,7 @@
 require 'spec_helper'
 require 'faraday'
 require 'typhoeus'
+require 'ostruct'
```

Comment:
## Reviewer note:

Tests were failing in this rspec because `#<NameError: uninitialized constant OpenStruct>`.

Need to import `OpenStruct` in this rspec file

Reply:
question: Did we determine the root cause for this? Why is this suddenly throwing an error now but not before? What changed?

Reply:
I am not sure why it suddenly throws an error; the tests run fine locally but not in CI, so I think it has to do with the environment differences. Perhaps a dependency in the CI got updated recently. I was surprised that an error was never thrown in the past, as you need to add `require ostruct` to use OpenStruct.